package.json

```diff
@@ -10,11 +10,14 @@
 	},
 	"main": "init.js",
 	"scripts": {
-		"api-dev": "NODE_PATH=./client:./ node init.js --watch ./server --watch ./utils --watch ./client --watch ./node_modules/@pubpub",
-		"api-prod": "NODE_PATH=./client:./ node init.js",
+		"api-dev": "npm run watch-server & NODE_PATH=./dist/server/client:./dist/server:./dist node init.js --watch ./dist/server --watch ./node_modules/@pubpub",
+		"api-prod": "NODE_PATH=./dist/server/client:./dist/server:./dist node init.js",
 		"build-dev": "webpack --config ./client/webpack/webpackConfig.dev.js --watch",
-		"build-dev-once": "rm -rf ./dist && webpack --config ./client/webpack/webpackConfig.dev.js",
-		"build-prod": "rm -rf ./dist && webpack --config ./client/webpack/webpackConfig.prod.js",
+		"build-dev-once": "rm -rf ./dist/client && webpack --config ./client/webpack/webpackConfig.dev.js && npm run build-server",
```

Comment:
**Important change** — we only nuke the `dist/client` instead of the entire `dist` folder so the TS build manifest isn't deleted every build (to take advantage of incremental builds).